### ISSUE TITLE
The colon(':') in the value of the list data type causes an error.

### ISSUE THREAD

**Issue by Lia-LV** [Osiris-Team/Dyml#32]
In a single String data type, the inclusion of colon in the value does not cause errors, 
but in the List data type, errors occur.

ex)
```
TestKey1: "TestValue1: abc"
TestListKey1:
  TestListKey2: "defaultTestValue2"
```
  
TestKey1 works fine without any problems, but TestListKey2 does not work properly.

**Comment by Osiris-Team** [comment]
@Lia-LV I'm not sure how dyml handles the parsing exactly, but I think it doesn't care for `"` and also removes those, which results in it actually thinking that `TestValue1` is a key and thus breaking your structure.
You can get more parsing related details be enabling debugging.

**Comment by Lia-LV** [comment]
@Osiris-Team Yes, That's right. 
```
TestKey1: "TestValue1: abc"
TestListKey1:
  - "TestValue2 : abc"
  - "TestValue3 : abcde"
```
I misheard the example in first comment. 
In the above yaml, the Key becomes "TestListKey1, TestValue2", and the value becomes "abc".
I think the data-type reading structure of the tree format affects the colon in the value of the List data-type format.
Even if I write it in List type, if there is a colon, it is recognized as Tree type data.

Can't you fix the bug by making if it's sub-data that starts with '-' recognized as a List or by using methods that modify the structure of the YamlReader?

**Comment by Osiris-Team** [comment]
@Lia-LV Yeah basically when it encounters a `-` it should read until the end of the line and set that as its value.
I thought it did that already, gonna check.

**Comment by Osiris-Team** [comment]
@Lia-LV should be fixed in 9.8.1
